publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceService.java

```diff
@@ -618,13 +618,12 @@ private List<TransactWriteItem> deleteResourceTransactionItems(List<Dao> daos) t
     }
 
     private void applyDeleteResourceConditions(TransactWriteItem deleteResource) {
-        Map<String, String> expressionAttributeNames = Map.of("#status", STATUS_FIELD_IN_RESOURCE, "#doi",
-                                                              DOI_FIELD_IN_RESOURCE);
+        Map<String, String> expressionAttributeNames = Map.of("#status", STATUS_FIELD_IN_RESOURCE);
```

Comment:
We now allow deleting non-published resource with doi. Deletion of drafted doi's will be handled automatically by event listener in doi-registrar.